plugin/db/driver.go

```diff
@@ -233,6 +233,11 @@ type MigrationInfo struct {
 	IssueID        string
 	Payload        string
 	CreateDatabase bool
+	// UseSemanticVersion is whether version is a semantic version.
```

Comment:
Let's give 2 full examples for UseSemanticVersion = true / false

Reply:
Done.